pyogrio/_err.pyx

```diff
@@ -173,8 +245,31 @@ cdef inline object exc_check():
         return
 
 
+cdef get_last_error_msg():
```

Comment:
This is not actually used yet? Above in the new code you directly use `CPLGetLastErrorMsg()`, but I suppose that could use this helper?

Reply:
True. I extracted the error message cleaning code in a new function `clean_error_message` and now use it where applicable.